examples/52_tanstack-router/src/server-time.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
```

Comment:
'use client' directive should be unnecessary. Does it cause any issues?

Reply:
is it not needed with the `useState` on this component?

Reply:
all components are client components, so it's not necessary.

Reply:
removed